CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,92 @@
+# FreeIPA Code of Conduct
+
+Our community is made up of a mixture of contributors from all over the world.
+We are diverse in our background, expertise or opinions and it is our
+strength, but diversity can also lead to communication issues and
+unhappiness. To that end, we have a few ground rules that we ask people to
+adhere to when operating in our space.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in
+the spirit in which it’s intended - a guide to make it easier to be excellent to
+each other:
+
+If you believe someone is violating the code of conduct, we ask that you report
```

Comment:
This is a very bad place to put this note to since the colon and the text above imply a list is about to follow. Why did you put it here?